### PR TITLE
[OSDOCS9863] Update Multi-architecture and Storage TP tables OCP 4.16 Release Notes

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-9863

Link to docs preview: 
https://77097--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-technology-preview-tables
Storage table 14/Multi-Architecture  table 17/Hosted Control Plane table 24

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by SNiemann15** [comment]
@lpettyjo fyi I updated the PowerVS entry in the Storage TP table but left the other rows unchanged

**Review by prb112 — APPROVED** [review]
/LGTM

@mjturek do you want to add an LGTM

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -1076,14 +1076,29 @@ In the following tables, features are marked with the following statuses:
...
+
+|Hosted control planes for {product-title} on {ibm-power-title}
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Hosted control planes for {product-title} on {ibm-z-title}
+|Technology Preview
+|Technology Preview
+|Technology Preview
```
Comment (lahinson) [inline release_notes/ocp-4-16-release-notes.adoc:1102]: @SNiemann15 Thanks so much for updating the hosted control planes table! We also need to add a row for non-bare metal agent machines:
```suggestion
|Hosted control planes for {product-title} using non-bare metal agent machines
|Not Available
|Technology Preview
|Technology Preview
```

**Comment by holgwolf** [comment]
lgtm

**Pull request merged**